community/kernel/src/main/java/org/neo4j/graphdb/factory/GraphDatabaseSettings.java

```diff
@@ -488,6 +488,12 @@
     @Description( "Log parameters for executed queries that took longer than the configured threshold." )
     public static final Setting<Boolean> log_queries_parameter_logging_enabled = setting( "dbms.logs.query.parameter_logging_enabled", BOOLEAN, TRUE );
 
+    @Description( "Log detailed time information for executed queries that took longer than the configured threshold." )
```

Comment:
Spoke @spacecowboy he mentioned that not everything is auto generated and we'll have to change the default config files here 
https://github.com/neo4j/neo4j/blob/3.2/packaging/standalone/standalone-community/src/main/distribution/text/community/conf/neo4j.conf

https://github.com/neo4j/neo4j/blob/3.2/packaging/standalone/standalone-enterprise/src/main/distribution/text/enterprise/conf/neo4j.conf